src/hooks/useBasisForm/validation/validateInput.ts

```diff
@@ -3,7 +3,7 @@ import { InternalInputProps } from "../../../components/Input/types";
 export const NUMERIC_REGEX = /^\d*$/;
 export const DECIMAL_REGEX = /^\d*(\.\d{2})?$/;
 
-const isInputEmpty = (value: string) => value.trim() === "";
+export const isInputEmpty = (value: string) => value.trim() === "";
```

Comment:
`!!value.trim()` can work too . Up to you mate

Reply:
It was like that from before. I don't mind either way